apis/fluentbit/v1alpha2/plugins/output/splunk_types.go

```diff
@@ -0,0 +1,133 @@
+package output
+
+import (
+	"fmt"
+
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins"
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins/params"
+)
+
+// +kubebuilder:object:generate:=true
+
+// OpenSearch is the opensearch output plugin, allows to ingest your records into an OpenSearch database.
+type Splunk struct {
+	// IP address or hostname of the target OpenSearch instance, default `127.0.0.1`
+	Host string `json:"host,omitempty"`
+	// TCP port of the target Splunk instance, default `8088`
+	// +kubebuilder:validation:Minimum:=1
+	// +kubebuilder:validation:Maximum:=65535
+	Port *int32 `json:"port,omitempty"`
+	// Specify the Authentication Token for the HTTP Event Collector interface.
+	SplunkToken string `json:"splunkToken,omitempty"`
+	//Buffer size used to receive Splunk HTTP responses: Default `2M`
+	// +kubebuilder:validation:Pattern:="^\\d+(k|K|KB|kb|m|M|MB|mb|g|G|GB|gb)?$"
+	BufferSize string `json:"bufferSize,omitempty"`
```

Comment:
bufferSize -> http_buffer_size

Reply:
ah, ok that's what you meant, will fix that :)